src/rust/cryptography-x509-verification/src/policy/mod.rs

```diff
@@ -488,6 +497,22 @@ impl<'a, B: CryptoOps> Policy<'a, B> {
             )));
         }
 
+        // CA/B 6.1.5: Key sizes
+        // NOTE: We don't currently enforce that RSA moduli are divisible by 8,
+        // since other implementations don't bother.
+        let issuer_spki = &issuer.certificate().tbs_cert.spki;
+        if matches!(
+            issuer_spki.algorithm.params,
+            AlgorithmParameters::Rsa(_) | AlgorithmParameters::RsaPss(_)
```

Comment:
Sanity checking myself: I can remove this `RsaPss` variant, right? I know it's a key type but AFAICT it never appears in SPKI AlgorithmIdentifiers.

Reply:
It can absolutely occur in an SPKI AlgorithmIdentifier.

But it'll never be true _here_ because it's not listed as a valid algorithm identifier which we check a few lines up.